app.js

```diff
@@ -1,75 +1,91 @@
 
 //Greeting
 var userName = prompt ('What\'s your name?');
-if(userName == null || userName === ""){
-  userName = "guest";
-}
+// if(userName == null || userName === ''){
+//   userName = 'guest';
+// }
+if (!userName) userName = 'guest';
```

Comment:
Nice work with this line. Good use of the "!" operator. To achieve the same effect, on line 3 you could have also written:

_var userName = prompt (What\'s your name?') || 'guest'._

That would cause the user's name to short circuit to guest if they provided no input to the prompt.